src/main/java/com/example/demo/models/paypal/OrderUnitRequestModel.java

```diff
@@ -0,0 +1,95 @@
+package com.example.demo.models.paypal;
+
+import java.math.BigDecimal;
+
+public class OrderUnitRequestModel {
+
+	public class Amount {
+		private String currency_code="USD";
```

Comment:
what happens when different currency is used?

Reply:
USD is used as a default for everything there is no option to change it in the application, if the payer uses something else it will convert to usd